.github/workflows/test.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       matrix:  # use latest available versions and be consistent on all workflows!
         include:
           - java_version: 11  # Last that can compile zipkin core to 1.6 for zipkin-reporter
-            maven_args: -Prelease -Dgpg.skip
+            maven_args: -Prelease -Dgpg.skip -Dmaven.javadoc.skip=true
```

Comment:
I forget if it's possible to wrote javadoc that fails to build, in which case this could allow unreleasable code to be merged. But if not an issue should be good

Reply:
It's a good point. deploy doesn't run the tests, but it does build javadoc. I'll raise a PR to add a job that just builds javadoc, so that way it is in parallel also.